diff --git a/node_modules/typescript-json-schema/dist/typescript-json-schema.js b/node_modules/typescript-json-schema/dist/typescript-json-schema.js
index feabae3..871906c 100644
--- a/node_modules/typescript-json-schema/dist/typescript-json-schema.js
+++ b/node_modules/typescript-json-schema/dist/typescript-json-schema.js
@@ -479,6 +479,9 @@ var JsonSchemaGenerator = (function () {
             else if (flags & ts.TypeFlags.Boolean) {
                 definition.type = "boolean";
             }
+			else if (flags & ts.TypeFlags.BigInt) {
+				definition.type = "integer";
+			}
             else if (flags & ts.TypeFlags.ESSymbol) {
                 definition.type = "object";
             }
